.circleci/config.yml

```
@@ -31,33 +31,34 @@ aliases:
       # Must be absolute path or relative path from working_directory
       at: /tmp/docker
 
-  - &install_openshift_cluster
+  - &install_k8s_requirements
     run:
-      name: Install the OC client
+      name: Install the kubectl client and k3d
       command: |
-        wget https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz -P /tmp/openshift/
-        tar xzf /tmp/openshift/openshift*.tar.gz --strip-components=1 -C /tmp/openshift/
-        cp /tmp/openshift/oc $HOME/bin/
+        curl -Lo "${HOME}/bin/kubectl" "https://dl.k8s.io/release/v1.20.2/bin/linux/amd64/kubectl"
+        curl -Lo /tmp/kubectl.sha256 "https://dl.k8s.io/v1.20.2/bin/linux/amd64/kubectl.sha256"
```

Should we declare a variable for this?
```suggestion
        export RELEASE="v1.20.2"
        curl -Lo "${HOME}/bin/kubectl" "https://dl.k8s.io/release/${RELEASE}/bin/linux/amd64/kubectl"
        curl -Lo /tmp/kubectl.sha256 "https://dl.k8s.io/${RELEASE}/bin/linux/amd64/kubectl.sha256"
```

---

Definitely! I introduced 2 new variables : `KUBECTL_RELEASE` and `K3D_RELEASE`